nucliadb/nucliadb/tasks/consumer.py

```diff
@@ -113,6 +137,10 @@ async def subscription_worker(self, msg: Msg):
             )
             try:
                 await self.callback(self.context, task_msg)  # type: ignore
+            except asyncio.CancelledError:
```

Comment:
Doubt: `CancelledError` can be raised here?

Reply:
yes! if the task nats consumer is finalized and there are running tasks, it will cancel them, and therefore we need to nack the msgs.